va/va.h

```diff
@@ -4835,7 +4835,17 @@ VAStatus vaDeassociateSubpicture (
  * brightness etc. in the rendering process.  The application can query what
  * attributes are supported by the driver, and then set the appropriate attributes
  * before calling vaPutSurface()
+ *
+ * Display attributes also are used to query/set display adaptor (vaDisplay) related information.
+ * These attributes do not depend on vaConfig, only related with platform or display adaptor
+ * They could not be used for vaPutSurface.
+ * App could call vaQueryDisplayAttributes/vaGetDisplayAttributes to query these attributes
+ * at anytime after vaInitialize, but only could call vaSetDisplayAttributes for these attributes
+ * after vaInitialize and before any other function call if the attributes is settable.
```

Comment:
Can we modify the behavior to:

App could call vaQueryDisplayAttributes/vaGetDisplayAttributes to query these attributes at anytime after vaInitialize,
but only could call vaSetDisplayAttributes for these attributes BEFORE vaInitialize.

Which make the vaInitialize a time point  to bind the tile?

Reply:
if we make the change just like your suggestion, it means that there are still no vaSetDisplayAttributes function pointer before initialize,  part of the logic should in libva instead of backend driver.

Reply:
OK, then I can accept this.